portal/views.py

```diff
@@ -103,6 +107,13 @@ class HomeView(View):
     template = "portal/home.html"
 
     def get(self, request, id=0):
+        logger.debug(
+            "This is a debug message",
+        )
+        logger.info("This is an info message")
+        logger.warning("This is a warning message")
+        logger.error("This is an error message")
+        logger.critical("This is a critical message")
```

Comment:
Maybe we put this in some startup script? Seems like it could have a use.

Reply:
Yeah sure! Can think about.